### DISCUSSION TITLE
Limit on Number of Projects?

### DISCUSSION THREAD

**Discussion by aarondfrancis** [vercel/community#253] [Help] [answered]
Hey y'all,

Is there a limit on the number of projects an account can create? The only limit I see that may speak to that is "Vercel Projects Connected per Git Repository," but that only has to do with git repos, which I'm not concerned about.

Thanks for any insight you might be able to share!

Aaron

**Comment by leerob** [comment, marked as answer]
Could you share more about what you're trying to build that could potentially have a large number of projects?

**Reply by aarondfrancis** [reply]
For sure! I created a library that allows Laravel developers to package, distribute, and execute AWS Lambda functions directly from their Laravel application. You can read more here: https://github.com/hammerstonedev/sidecar or here https://m.youtube.com/watch?v=2UyDBArGLDY&t=161.

Over the weekend I got it working with Vercel too, which is amazing because a lot of folks don't want to muck about with AWS stuff. 

As you know, in the Lambda world, everything is isolated per Lambda, so things like environment variables, libraries, and other resources are all disparate. And individual functions can be deployed individually. I'm looking to recreate that in Vercel, but the only way I can see to do that is via projects. Each project would be pretty small, and they'd all be managed via Sidecar through Vercel's API. 

If you have a better idea I'm all ears! The projects method works perfectly, I just don't want my users to hit some limit I'm unaware of. Always happy to chat further too.

**Reply by leerob** [reply]
No limits on the number of projects from what I can tell 😄
